sites/geohub/package.json

```diff
@@ -134,10 +135,13 @@
 		"@azure/service-bus": "^7.9.3",
 		"@azure/storage-blob": "^12.17.0",
 		"@azure/web-pubsub": "^1.1.1",
+		"@carbon/charts-svelte": "^1.15.7",
 		"@mapbox/geo-viewport": "^0.5.0",
 		"@mapbox/vector-tile": "^1.3.1",
 		"@undp-data/style": "^2.0.1",
 		"crypto-js": "^4.2.0",
+		"d3-cloud": "^1.2.7",
+		"d3-sankey": "^0.12.3",
```

Comment:
@mikedstr Could you move these three frontend libraries to `devDependencies`? (but I think we should remove dependencies of carbon design system (@carbon/charts-svelte), see my other comment).

Reply:
@JinIgarashi Carbon Charts is actually a separate package from Carbon Design System components. It is a d3 wrapper which purpose is solely for data visualization. So, I don't think it will cause conflicts with Bulma and UNDP styles. I also showed it to Ioan last Friday and he's okay with it.

Reply:
@JinIgarashi Okay, I'll remove the `vega` devDependencies

Reply:
@mikedstr ok, got it. please just move frontend libraries to `devDependencies`. If you include them in `dependencies`, the size of node_modules in production will be increased.